devtools/common/src/main/filtered/extensions.json

```diff
@@ -700,5 +700,21 @@
     "groupId": "io.quarkus",
     "artifactId": "quarkus-vertx",
     "guide": "https://quarkus.io/guides/using-vertx"
+  },
+  {
+    "name": "Narayana STM - Software Transactional Memory",
+    "labels": [
+      "narayana-stm",
+      "narayana",
+      "stm",
+      "transactions",
+      "transaction",
+      "software-transactional-memory",
+      "tx",
+      "txs"
+    ],
+    "groupId": "io.quarkus",
+    "artifactId": "quarkus-narayana-stm",
+    "guide": "https://quarkus.io/guides/stm-guide"
```

Comment:
Sorry, missed that the first time but this file is mostly alphabetically sorted too so it would be nice to add this one together with Narayana.

And that will be it for me :).

Thanks!